osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-bundle.xsd

```diff
@@ -538,6 +553,7 @@
         <xsd:element ref="tns:AssociationLnObjectsResponse"/>
         <xsd:element ref="tns:GetConfigurationObjectResponse"/>
         <xsd:element ref="tns:GetPowerQualityProfileResponse"/>
+        <xsd:element ref="tns:ActualPowerQualityResponse"/>
```

Comment:
a bit of a "problem" here is that this name does not follow the Get pattern. All Get actions are preceded with Get, and this one is not. Consider changing it for coherence.

Reply:
One on three of the Response names do no start with a verb ("Get") while their corresponding Requests do. Making it coherent should be done all together.